apps/marketplace/components/BuyerATM/BuyerATMAdditionalInformationStage.js

```diff
@@ -2,19 +2,45 @@ import React, { Component } from 'react'
 import PropTypes from 'prop-types'
 import { connect } from 'react-redux'
 import { actions, Form } from 'react-redux-form'
-import { required, validPhoneNumber, dateIs2DaysInFuture } from 'marketplace/components/validators'
 import formProps from 'shared/form/formPropsSelector'
 import Textfield from 'shared/form/Textfield'
 import AUheading from '@gov.au/headings/lib/js/react.js'
+import format from 'date-fns/format'
+import addDays from 'date-fns/add_days'
+import {
+  required,
+  validPhoneNumber,
+  validDate,
+  dateIs2DaysInFuture,
+  dateIsBefore
+} from 'marketplace/components/validators'
 import ErrorAlert from 'marketplace/components/Alerts/ErrorAlert'
-import ClosingDateControl from 'marketplace/components/BuyerBriefFlow/ClosingDateControl'
-import styles from './BuyerATMAdditionalInformationStage.scss'
+import DateControl from 'marketplace/components/BuyerBriefFlow/DateControl'
+import styles from 'marketplace/main.scss'
 
-class BuyerATMAdditionalInformationStage extends Component {
+const requiredContactNumber = v => required(v.contactNumber)
+const contactNumberFormat = v => validPhoneNumber(v.contactNumber)
+const requiredClosedAt = v => required(v.closedAt)
+const closedAtIsValid = v => validDate(v.closedAt)
+const closedAtIs2DaysInFuture = v => !closedAtIsValid(v) || dateIs2DaysInFuture(v.closedAt)
+const closedAtIsBefore = v => !closedAtIsValid(v) || dateIsBefore(v.closedAt, addDays(new Date(), 364))
```

Comment:
Why are these changes here?  They look like they might be from another one of your pull requests.